gradoop-common/src/main/java/org/gradoop/common/model/impl/id/GradoopIds.java

```diff
@@ -182,11 +185,14 @@ public boolean containsAll(GradoopIds ids) {
   /**
    * Checks if the specified ids are contained in the set.
    *
-   * @param ids the ids to look for
+   * @param other the ids to look for
    * @return true, iff all specified ids are contained in the set
    */
-  public boolean containsAll(Collection<GradoopId> ids) {
-    for (GradoopId id : ids) {
+  public boolean containsAll(Collection<GradoopId> other) {
```

Comment:
We should change to scope to `Set<GradoopId>` here to avoid strange corner cases (i.e. having duplicate ids in the argument which are actually contained in the current set.

Reply:
This would change the signature of the class. If we do that, why not just inherit from `AbstractCollection`? The class pretty much is a collection anyway, so it might be nice to use a familiar abstraction.

Reply:
Sounds reasonable